incubator/drone/values.yaml

```diff
@@ -0,0 +1,98 @@
+## Drone
+## ref: https://hub.docker.com/r/drone/drone
+##
+image: drone/drone:0.5
+imagePullPolicy: Always
+
+## For minikube, set this to NodePort, elsewhere use LoadBalancer
+## ref: http://kubernetes.io/docs/user-guide/services/#publishing-services---service-types
+##
+serviceType: LoadBalancer
+
+# if not set autogen
+#shared_secret: ""
+
+# See http://readme.drone.io/0.5/manage/server/ for possible values here.
+server:
+  open: true
+  admin: ""
+  orgs: ""
+
+  ## Configure resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources:
+    requests:
+      memory: 32Mi
+      cpu: 40m
+    limits:
+      memory: 2Gi
+      cpu: 1
+  database:
+    driver: sqlite3
+    config: /var/lib/drone/drone.sqlite
+
+  #See http://readme.drone.io/admin/ for possible values here.
+  remote:
+    github:
```

Comment:
IMHO there should be reference github http://readme.drone.io/admin/setup-github/ because lots of folks would likely use that section.